README.md

```diff
@@ -1,2 +1,3 @@
-# relativepath.tech
-The RelativePath Website
+# React + TypeScript + Vite
```

Comment:
Need more information on dependencies, platform, and would be great to see instructions on how to get setup and run the site.

Reply:
`README.md` has been updated with usage instructions.